docs/VSCode.md

```diff
@@ -0,0 +1,48 @@
+# Using VS Code for Homebrew Development
+
+> **Note:** This guide assumes you are using `rbenv` to manage your Ruby version.
+> However, the Ruby LSP is compatible with other Ruby version managers.
+> Check the [Ruby LSP documentation](https://github.com/Shopify/ruby-lsp/tree/main/vscode) for more information.
+
+## Prerequisites
+
+- [VS Code](https://formulae.brew.sh/cask/visual-studio-code)
+  - `brew install --cask visual-studio-code`
+- [`rbenv`](https://formulae.brew.sh/formula/rbenv)
+  - `brew install --formula rbenv`
+
+## Setup `rbenv`
+
+Once `rbenv` is set up correctly (see the [installation instructions](https://github.com/rbenv/rbenv#installation)), install the correct Ruby version:
+
+```bash
+rbenv install "$(cat "$(brew --repository)/.ruby-version")"
+```
```

Comment:
Would recommend doing a `brew install ruby@X` and `brew rbenv-sync` here instead.